src/ziggurat/init.clj

```diff
@@ -188,9 +195,9 @@
 (defn- validate-routes-against-config
   ([routes route-type]
    (doseq [[topic-entity handler-map] routes]
-     (let [route-config (-> (ziggurat-config)
-                            (get-in [route-type topic-entity]))
-           channels      (-> handler-map (dissoc :handler-fn) (keys) (set))
+     (let [route-config    (-> (ziggurat-config)
```

Comment:
why `->` if only one function being called?

Reply:
For clarity I believe